src/templates/index.jsx

```diff
@@ -0,0 +1,77 @@
+
+import React, { useState } from 'react';
+import Helmet from 'react-helmet';
+import { useDispatch, useSelector } from 'react-redux';
+
+import Navigation from '~/components/molecules/navigation';
+import ArtefactOverview from '~/components/organisms/artefact-overview';
+import SearchOverview from '~/components/organisms/search-overview';
+
+import i18n from '~/i18n';
+
+import {
+  searchFor,
+  getSearchTerm,
+  getSearchLoading,
+  getSearchResultItems,
+} from '~/features/globalSearch/globalSearchSlice';
+
+
+export default ({ pageContext }) => {
+  const { lang, graphics } = pageContext;
+  i18n(lang.code);
+
+  const [currentArtefactView, setCurrentArtefactView] = useState(ArtefactOverview.DefaultView);
+  const dispatch = useDispatch();
+  const searchTerm = useSelector(getSearchTerm);
```

Comment:
Aufruf von `useSelector` mit einer Selektor-Funktion, um in diesem Fall an den Suchstring zu kommen, der im Store liegt (GlobalSearch).

Reply:
Wird geschlossen, da Redux durch MobX ausgetauscht wurde.